Breeze.TumbleBit.Client/TumbleBitManager.cs

```diff
@@ -822,6 +852,31 @@ public int LastBlockTime
                 }
             }
         }
+
+        private static string TimeSpanInWordFormat(double fromHours)
+        {
+            var timeSpan = TimeSpan.FromHours(fromHours);
+
+            var days = timeSpan.Days.ToString();
+            var hours = timeSpan.Hours.ToString();
+
+            var formattedTimeSpan = string.Empty;
+
+            if (timeSpan.Days > 0)
+            {
+                formattedTimeSpan = days + " days";

```

Comment:
Would it be better to have string interpolation here and the following few lines

Reply:
done